dart/lib/src/protocol/sentry_span.dart

```diff
@@ -94,15 +105,25 @@ class SentrySpan extends ISentrySpan {
   ISentrySpan startChild(
     String operation, {
     String? description,
+    DateTime? startTimestamp,
   }) {
     if (finished) {
       return NoOpSentrySpan();
     }
 
+    if (startTimestamp?.isBefore(_startTimestamp) ?? false) {
```

Comment:
Same issue from https://github.com/getsentry/sentry-dart/pull/676/files#r800678030
are `startTimestamp` and `_startTimestamp` already or not?